### PR TITLE
Format / Style changes

### PR DESCRIPTION
**Pull request by leb2**
Mostly minor formatting changes, and a bug fix. Sorry, it's kind of random.

### CODE REVIEW

**Comment by fouric** [comment]
Nice!  Most of these will would make the code-base much easier to read and more correct.  However, I have one small issue with this: the "delete last character in search buffer" change.  In DSK, the dash key is one key right of the right pinky, requiring a much smaller stretch than going all the way to backspace.  It might make more sense to merely create an entry in the keybindings file that can be mapped to something more convenient for QWERTY users (such as forward slash) then use backspace, which would both be a longer reach and require special handling code because it can't be represented as a character.

How can we change this?

**Comment by leb2** [comment]
Can it be so that the backspace key works as normal, but also have another changeable keybinding for delete? This should work assuming that the backspace key is not needed for something else.

**Comment by fouric** [comment]
Yes, it would technically work, but it's more of a style issue.  The "style" of Lightning is to have as few redundant keybindings as possible, and to move those keybindings as close to the default finger positions as possible.  Would you be open to just leaving "delete" as "-" (or whatever the user configures it to be) and maintaining a separate branch with separate bindings for enter, backspace, etc?

File: lightning-cd.py [hunk 1]
```diff
@@ -79,114 +85,113 @@ def drawFileList(t, ystart, yend, mode, selected):# {{{
...
+                    runCommandOnFile(os.path.realpath('.'), fileBrowser + ' ' + os.path.realpath('.') + ' > /dev/null 2>&1')
+                elif letter == keybindings.KEY_TMUX:
+                    runCommandOnFile(os.path.realpath('.'), 'tmux > /dev/null')
+            elif mode == Mode.SEARCH:
+                if letter:
+                    if letter in charRange:
+                        searchBuffer = searchBuffer + letter
+                    elif letter == keybindings.KEY_DELETE:
+                        searchBuffer = searchBuffer[:-1]
+                elif keycode == 127: # DELETE
```
Comment (fouric) [inline lightning-cd.py]: This, if actually used in the code (I would prefer to just leave it as a user-definable printable character), should probably be termbox.TB_KEY_BACKSPACE2 (there's a list of all constants in termbox.h)

Comment (fouric) [inline lightning-cd.py]: This line in particular, I mean.

**Comment by leb2** [comment]
Yeah, that makes sense.

**Pull request merged**